com.reprezen.swagedit.openapi3.tests/src/com/reprezen/swagedit/openapi3/schema/SchemaValidationTest.xtend

```diff
@@ -0,0 +1,62 @@
+package com.reprezen.swagedit.openapi3.schema
+
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.dataformat.yaml.YAMLMapper
+import com.google.common.collect.Iterators
+import com.google.common.collect.Lists
+import com.reprezen.swagedit.core.validation.ErrorProcessor
+import com.reprezen.swagedit.core.validation.SwaggerError
+import com.reprezen.swagedit.core.validation.Validator
+import com.reprezen.swagedit.openapi3.Activator
+import java.net.URL
+import java.util.Collection
+import java.util.Iterator
+import java.util.Set
+import org.eclipse.core.runtime.Platform
+import org.junit.runner.RunWith
+import org.junit.runners.Parameterized
+import org.junit.runners.Parameterized.Parameter
+import org.junit.runners.Parameterized.Parameters
+
+import static org.junit.Assert.*
+import org.junit.Test
+
+@RunWith(typeof(Parameterized))
+class SchemaValidationTest {
+
+	@Parameters(name = "{index}: {1}")
+	def static Collection<Object[]> data() {
+		val Iterator<URL> specFiles = Iterators::forEnumeration(
```

Comment:
Not an issue but It needs to be run as junit plugin test, when I try to run it as normal junit test I get

```
java.lang.NullPointerException
	at com.reprezen.swagedit.openapi3.schema.SchemaValidationTest.data(SchemaValidationTest.java:39)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.runners.Parameterized.allParameters(Parameterized.java:292)
	at org.junit.runners.Parameterized.<init>(Parameterized.java:282)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at org.junit.internal.builders.AnnotatedBuilder.buildRunner(AnnotatedBuilder.java:29)
	at org.junit.internal.builders.AnnotatedBuilder.runnerForClass(AnnotatedBuilder.java:21)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)
	at org.junit.internal.builders.AllDefaultPossibilitiesBuilder.runnerForClass(AllDefaultPossibilitiesBuilder.java:26)
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)
	at org.junit.internal.requests.ClassRequest.getRunner(ClassRequest.java:26)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestLoader.createUnfilteredTest(JUnit4TestLoader.java:84)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestLoader.createTest(JUnit4TestLoader.java:70)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestLoader.loadTests(JUnit4TestLoader.java:43)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:444)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
```